changes/6144.added

```diff
@@ -1 +1 @@
-Added Virtual Device Context model and API.
\ No newline at end of file
+Added Virtual Device Context and InterfaceVDCAssignment model and API.
```

Comment:
This is probably fine to be a new change fragment (6211.added) rather than an addition to that one.